tests/integration/test_tiling.py

```diff
@@ -18,7 +25,10 @@
 from otx.core.data.entity.detection import DetBatchDataEntity
 from otx.core.data.entity.tile import TileBatchDetDataEntity
 from otx.core.data.module import OTXDataModule
+from otx.core.model.entity.detection import OVDetectionModel
+from otx.core.model.entity.instance_segmentation import OVInstanceSegmentationModel
 from otx.core.types.task import OTXTaskType
+from otx.engine import Engine
 
 
 class TestOTXTiling:
```

Comment:
Similar to Vinnam's comment, wouldn't it be appropriate for us to move most of this to unit-test and for integration test, test the tiling recipe in the existing e2e (Do we need to have separate tests?)?

Reply:
CLI tests currently cover tile recipes, however, engine doesn't cover those recipes and I added a new test in engine API test.